packages/keeper_security_siem_integration/manifest.yml

```
@@ -0,0 +1,31 @@
+format_version: 3.0.0
+name: keeper
+title: "Keeper Security"
+version: 1.0.0
+description: >
+  Keeper Security agentless integration for collecting audit events directly  via Elasticsearch Bulk API. No agents required - Keeper pushes data directly  to Elasticsearch ingest pipelines.
```

There are double spaces.

```suggestion
  Keeper Security agentless integration for collecting audit events directly via Elasticsearch Bulk API. No agents required - Keeper pushes data directly to Elasticsearch.
```

Does this description render fully or is it truncated? It's longer than most packages' descriptions.